graphql/database/missions/utils.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-nested-ternary */
+/* eslint-disable no-param-reassign */
```

Comment:
Move this to the specific line below. IMO, reducing is a fine case to disable this rule.